packages/cli-exec/src/exec.js

```diff
@@ -110,4 +110,41 @@ async function* spawn(cmd, args) {
   }
 }
 
+const autoStart = command('start', {
```

Comment:
we don't need to add a new command here. 

in the callback for  `commad('exec',...)` we'll update the value of 

`percy.projectType, percy.skipDiscovery` based on the token prefix ( have this token logic in package which is required by all `cli-*` commands since we'll need a check in future in those packages.

Reply:
Changed this as suggested.